install.sh

```diff
@@ -1014,22 +1014,17 @@ EOS
 ohai "Next steps:"
 case "${SHELL}" in
   */bash*)
-    if [[ -r "${HOME}/.bash_profile" ]]
-    then
-      shell_profile="${HOME}/.bash_profile"
-    else
-      shell_profile="${HOME}/.profile"
-    fi
+    shell_rcfile="${HOME}/.bashrc"
     ;;
   */zsh*)
-    shell_profile="${HOME}/.zprofile"
+    shell_rcfile="${HOME}/.zshrc"
     ;;
   *)
-    shell_profile="${HOME}/.profile"
+    shell_rcfile="${ENV:-"${HOME}/.profile"}"
```

Comment:
Reference for the `ENV` environment variable:

[Stack Overflow: Is there an alternative for .bashrc for /bin/sh?](https://stackoverflow.com/questions/39621880/is-there-an-alternative-for-bashrc-for-bin-sh)